utilities/backupable/backupable_db.cc

```diff
@@ -994,23 +993,26 @@ Status BackupEngineImpl::Initialize() {
     directories.emplace_back(meta_path, &meta_directory_);
     // create all the dirs we need
     for (const auto& d : directories) {
-      auto s = backup_env_->CreateDirIfMissing(d.first);
-      if (s.ok()) {
-        s = backup_env_->NewDirectory(d.first, d.second);
+      IOStatus io_s = backup_env_->GetFileSystem()->CreateDirIfMissing(
+          d.first, IOOptions(), nullptr);
```

Comment:
Nit: Move IOOptions into a variable outside of the loop.  This is currently creating two IOOptions every time through the loop.

Reply:
Should there be a class variable IOOptions?